dataservice/api/participant/models.py

```diff
@@ -13,12 +13,18 @@ class Participant(db.Model, Base):
 
     :param kf_id: Unique id given by the Kid's First DCC
     :param external_id: Name given to participant by contributor
+    :param family_id: Id for the participants grouped by family
+    :param is_proband: Is participant is proband of study
+    :param consent_type: Type of the consent participant belongs to
     :param created_at: Time of object creation
     :param modified_at: Last time of object modification
     """
     __tablename__ = "participant"
 
     external_id = db.Column(db.Text(), doc='ID used by external study')
+    family_id = db.Column(db.Text())
+    is_proband = db.Column(db.Boolean())
```

Comment:
Hmm, I wonder if this should have a default, or at least be non-nullable?

Reply:
Yea I think it should have a default. When I had it on the StudyParticipant table, I had set the default to false because I assumed there would be more non-probands then probands? Idk if that's the correct assumption though. @allisonheath thoughts?

Reply:
I think non-nullable makes sense. I would be a bit wary about setting a default - there are downstream consequences to it being set wrong as there's several search use cases that focus specifically on probands and their families. But if you really want a default, I think there's slightly less impact if it's set to false when it's supposed to be true versus the other way around